memcached-image/config.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# that's what gets extracted from the archive (see the .kiwi file)
+# the archive itself is created by the OBS services (see _service)
+ln -s /memcached-docker-entrypoint/docker-entrypoint.sh /usr/local/bin/
```

Comment:
Is this meant to be the root path?

Reply:
yes. that's where it gets extracted by the <archive> parameter. And the tarball generation via a source service works that way (I didn't found a way to change that yet)